src/test/java/com/mavenit/argos/SmokeTestSuite.java

```diff
@@ -0,0 +1,286 @@
+        System.out.println("Product in the Basket is:   " + actual);
+    }
+
+    @Test
+    public void basketMultipleQuantityTest() throws InterruptedException {
+
+        doSearch("tripods, monopods and cases");
+        Thread.sleep(4000);
+        List<WebElement> productWebElements = driver.findElements(By.cssSelector("a[data-test='component-product-card-title']"));
+        List<WebElement> price = driver.findElements(By.cssSelector("div[class^=\"ProductCardstyles__PriceText-l8f8q8-14\"]"));
+        if (productWebElements.size() == 0) {
+            Assert.fail("No Products found with: " + "tripods, monopods and cases");
+        }
+        // TODO: 2020-02-08 this will be converted in future
+        Random random = new Random();
+        int randomNumber = random.nextInt(productWebElements.size()-1 );
+
+        WebElement selectedElement = productWebElements.get(randomNumber);
+        String selectedProductName = selectedElement.getText();
+        String Price= price.get(randomNumber).getText().replace("£", "");
+        double selectedProductPrice = Double.parseDouble(Price);
+
+        selectedElement.click();
+        Thread.sleep(2000);
+
+        String selectedQty = quantityToSelect("5");
+        double qtySelected = Double.parseDouble(selectedQty);
+        double totalPrice = selectedProductPrice*qtySelected;
+        String totalPriceOfSelectedProducts = String.format("%.2f", totalPrice);
+        Thread.sleep(2000);
+
+        addToBasket();
+        Thread.sleep(4000);
+
+        goToBasket();
+
+        Thread.sleep(2000);
+        String actualName = getProductNameInBasket();
+        String actualPrice = getProductPriceInBasket();
+        String totalPriceOfBasketProducts = actualPrice.replace("£", "");
+
+        assertThat(actualName, is(equalToIgnoringCase(selectedProductName)));
+        assertThat(totalPriceOfSelectedProducts,is(equalToIgnoringCase(totalPriceOfBasketProducts)));
+
+        System.out.println("\n\n'basketMultipleQuantityTest' Test Passed: Actual selected product & the product in the basket is same and total price of selected products are matching");
+
+        System.out.println("\nTotal price of Selected Products: £"+totalPriceOfSelectedProducts);
+        System.out.println("Total price of Basket Products: £" +totalPriceOfBasketProducts);
+
+        System.out.println("\nSingle price of each product is= £"+selectedProductPrice);
+        System.out.println("The quantity selected is: "+qtySelected);
+
+        System.out.println("\nTotal number of products are: " + productWebElements.size());
+        System.out.println("Index number of random product is : " + randomNumber);
+
+        System.out.println("\nActual selected product is: " + selectedProductName );
+        System.out.println("Product in the Basket is:   " + actualName);
+    }
+
+
+    @Test
+    public void basketMultipleProductTest() throws InterruptedException {
+        int numberOfProductsBuying = 2;
```

Comment:
2 is really small number , don’t you thing it will waste memory by storing in Int data type

Reply:
my mistake,, now data type has been changed to byte,, in build and working successfully as expected